src/data-loaders/RelationalDataLoader.js

```diff
@@ -75,8 +77,7 @@ const convertSQLResponseToRDSRecords = (rows) => {
 const convertPostgresSQLResponseToColumnMetaData = (rows) => {
   return rows.map((row) => {
     const typeName = Object.keys(pgTypes.builtins)
-      .find((d) => pgTypes.builtins[d] === row.dataTypeID)
-      .toUpperCase();
+      .find((d) => pgTypes.builtins[d] === row.dataTypeID);
```

Comment:
typeName may be `undefined`.

Reply:
Ok, but what happens to `toUpperCase`?

You can chain `find` with `?` to account for undefined maybe?